server/plugin/plugin.go

```diff
@@ -81,6 +87,48 @@ func (p *MatterpollPlugin) checkServerVersion() error {
 	return nil
 }
 
+// ensureBotAccount checks if the bot account exists and creates him if doesn't
+func (p *MatterpollPlugin) ensureBotAccount() error {
+	botUserID, err := p.Store.Bot().GetID()
+	if err != nil {
+		return errors.Wrap(err, "failed to get bot user from store")
+	}
+	if botUserID == "" {
+		bundlePath, err := p.API.GetBundlePath()
+		if err != nil {
+			return errors.Wrap(err, "failed to get bundle path")
+		}
+
+		profileImage, err := ioutil.ReadFile(filepath.Join(bundlePath, "assets", "logo_dark.png"))
+		if err != nil {
+			return errors.Wrap(err, "failed to read profile image")
+		}
+
+		bot := &model.Bot{
+			Username:    "matterpoll",
+			DisplayName: "Matterpoll",
+			Description: "Some description",
```

Comment:
Ideas for the wording?

Reply:
FYI: The NPS plugins uses `Created by the Net Promoter Score plugin.` (https://github.com/mattermost/mattermost-plugin-nps/blob/master/server/activate.go#L69)

But I think I like `Poll plugin for Mattermost` more. Or `Poll Bot`?

Reply:
`Poll Bot` sounds best for me 👍